docs/contributions/rfc_process.md

```diff
@@ -0,0 +1,115 @@
+6. The meeting may approve the RFC, reject it, or require changes before it can be considered again. Approved RFCs will be updated in Github by the sponsor.
+
+
+## RFC participants
+
+Many people are involved in the RFC process:
+
+
+
+*   **RFC author** — one or more community members who write an RFC and are committed to championing it through the process
+*   **RFC sponsor** — a maintainer or product manager who sponsors the RFC and will shepherd it through the RFC review process
+*   **review committee** — a group of maintainers who have the responsibility of recommending the adoption of the RFC.  This group can be emailed by emailing cirq-maintainers@googlegroups.com. This is typically the attendees at the Cirq Cync meeting.
+*   Any **community member** may help by providing feedback on whether the RFC will meet their needs.
+
+
+### RFC sponsors
+
+A sponsor is a project maintainer responsible for ensuring the best possible outcome of the RFC process. This includes:
+
+
+
+*   Advocating for the proposed design.
+*   Guiding the review committee to come to a productive consensus.
+*   If the RFC moves to implementation:
+    *   Ensuring proposed implementation adheres to the design.
+    *   Coordinate with appropriate parties to successfully land implementation.
+
+
+### RFC review committees
+
+The review committee decides on a consensus basis whether to approve, reject, or request changes. They are responsible for:
+
+
+
+*   Ensuring that substantive items of public feedback have been accounted for.
+*   Adding their meeting notes as comments to the PR.
+*   Providing reasons for their decisions.
+
+The constitution of a review committee may change according to the particular governance style and leadership of each project. For core TensorFlow, the committee will consist of contributors to the TensorFlow project who have expertise in the domain area concerned.
+
+
+## Implementing new features
+
+Once an RFC has been approved, implementation can begin.
+
+If you are working on new code to implement an RFC:
+
+
+
+*   Make sure you understand the feature and the design approved in the RFC. Ask questions and discuss the approach before beginning work.
+*   New features must include new unit tests that verify the feature works as expected. It's a good idea to write these tests before writing the code.
+*   Add or update relevant API documentation. Reference the RFC in the new documentation.
+*   Follow any other guidelines laid out in the CONTRIBUTING.md file in the project repo you're contributing to.
+*   Run unit tests before submitting your code.
+*   Work with the RFC sponsor to successfully land the new code. This could include PR / marketing of the new feature as well.
+
```

Comment:
It would probably be useful to point out that there is a specific deprecation process for code in Cirq.  This can be done by pointing people to the deprecation code in `_compat.py`.